.travis.yml.dist

```diff
@@ -28,6 +28,8 @@ env:
     # add composer's global bin directory to the path
     # see: https://github.com/drush-ops/drush#install---composer
     - PATH="$PATH:$HOME/.composer/vendor/bin"
+    # force composer 1.8+ to use a specific folder as home
+    #- export COMPOSER_HOME="$HOME/.composer/"
```

Comment:
I don't believe `export` is needed. Also is there any reason to keep it commented out?

Reply:
You're right let's change this ! 👍